### ISSUE TITLE
Add Stratum Build

### ISSUE THREAD

**Issue by toganlabs** [NetworkGradeLinux/meta-mion#73]
We've been requested to add a meta-stratum with images.

**Comment by johntoomey** [comment]
more info here https://opennetworking.org/stratum/

**Comment by johntoomey** [comment]
- Recipe for stratum
  - requires Bazel build system https://github.com/Wind-River/meta-tensorflow maybe?
  - Dependencies - packages etc listed here: https://github.com/stratum/stratum/blob/master/Dockerfile.build
  - Depends on Barefoot SDE

Requires ONLPv2 but we can possibly use the SDE directly

**Comment by N-Mc-Nally** [comment]
@toganlabs - queried in bug scrub by @CommitThis - is this the most accurate naming/labelling for this issue, based on timing of the next release.

**Comment by johntoomey** [comment]
all the code for this is in the new repo - https://github.com/NetworkGradeLinux/meta-mion-stratum - this needs to be tested during the release cycle..

**Comment by johntoomey** [comment]
here is the current state of this task: 
- the meta-mion-stratum layer has been updated to the 21.03 release which _appears_ to install and work correctly - 
- we are struggling to test this due to a lack of available hardware - the bf2556 we have access to isnt a good fit and we cannot get access to a bf6064 (or anything else for that matter) so i am putting this task on hold for the moment
- @toganlabs is going to follow up and see if we can get access to a bf6064 or something else to get this working

**Comment by johntoomey** [comment]
The stratum repo is no public and this appears to be working - testing is blocked by this issue https://github.com/NetworkGradeLinux/meta-mion-sde/issues/12 but in going to close this out anyway.
